### ISSUE TITLE
GPU fail

### ISSUE THREAD

**Issue by milenamilka755** [intel/openvino-plugins-ai-audacity#82]
I use https://github.com/Const-me/Whisper and works OK with my old GTX660.

If I set GPU in openvino-plugins-ai-audacity -FAIL and I must use (slow) CPU.

**Comment by RyanMetcalfeInt8** [comment]
Hi @milenamilka755,

Can you give some more details about your hardware setup? What version of CPU, etc. -- And I assume the GPU in this case is the GTX660?

Thanks,
Ryan

**Comment by milenamilka755** [comment]
Win10, 32 Ram, CPU Ryzen 3700, GPU Nvidia GTX 660 2GB

**Comment by RyanMetcalfeInt8** [comment]
Thanks. I've heard that OpenVINO may support certain NVidia GPUs, although even in the case where they are functional, I don't think they are too performant as OpenVINO's GPU backed uses OpenCL.

From the point of view of this plugin, we pass a model to the OpenVINO framework, and the device string ("GPU" in this case). It's up to OpenVINO to 'do the right thing' -- so there's not too much that can be done at the level of this plugin. 

TL;DR this issue is more about what OpenVINO supports, rather than what this plugin supports. Closing as not planned.
